Cargo.toml

```diff
@@ -12,7 +12,6 @@ include = [
   "README.md",
   "CHANGELOG.md",
   "LICENSE",
-  "config_spec.toml",
```

Comment:
Please open a separate PR for this one as well.

Reply:
I will too. I am just pushing things in this PR at the moment. Will split into separate PRs later